### DISCUSSION TITLE
Bootloader for Honor Play (COR-AL00) Kirin 970.

### DISCUSSION THREAD

**Discussion by Abhilash-Jarvis** [kitsuned/PotatoNV#124] [Q&A] [answered]
I needed bootloader unlock method for Kirin 970 for honor play, please provide one. I'm really in need for this.
Honor Play
COR-AL00
Kirin 970
EMUI 9.1.0.396(C675E4R1P9)
I will give all the other details for the same.
YouBcontact me on abhilash.in7+potatoes@gmail.com

**Comment by mashed-potatoes** [comment, marked as answer]
try to read README.

**Reply by Madhavan0731** [reply]
can you please send me too. my email: barryalen3107@gmail.com
